flow.py

```diff
@@ -39,7 +39,10 @@ def wake(self):
 
     def send_a_packet(self):
         if (self.amount > 0):
-            packet = PayloadPacket(self.source, self.destination)
+            # numbers the packets in descending order
+            # packet is uniquely identified by flow and packet number
+            packetID = "P" + str(self.amount / 1024) + self.identifier
```

Comment:
I think it'd be good to not hardcode 1024 here (or elsewhere). Any way we can avoid this?